coral-component-list/src/scripts/SelectList.js

```diff
@@ -431,13 +404,39 @@ class SelectList extends BaseComponent(HTMLElement) {
    @private
    */
   _resetTabTarget(forceFirst = false) {
-    let items = this.items._getAllSelected().filter(item => !item.hasAttribute('hidden'));
+    let items = this.items._getAllSelected().filter(item => !item.hasAttribute('hidden') && item.offsetParent);
 
     if (items.length === 0) {
-      items = this.items._getSelectableItems().filter(item => !item.hasAttribute('hidden'));
+      items = this.items._getSelectableItems().filter(item => !item.hasAttribute('hidden') && item.offsetParent);
     }
     
-    this._tabTarget = forceFirst ? items[0] : (items.find(item => item.tabIndex === 0) || items[0]); 
+    this._tabTarget = forceFirst ? items[0] : (items.find(item => item.tabIndex === 0) || items[0]);
+
+    if (this.groups && this._groups.getAll().length > 0) {
```

Comment:
if this condition fails (if (this.groups && this._groups.getAll().length > 0)) 
we aren't setting tabindex = 0 on this. _tabTarget
Shouldn't we set it?

Reply:
Looks like we are detecting last active/selected item by condition : items.find(item => item.tabIndex === 0) 

We need to ensure that we always set  item.tabIndex = 0 when we focus on an item.